vm/src/utils.rs

```diff
@@ -189,17 +193,19 @@ pub mod test_utils {
             }
         };
     }
-    pub(crate) use memory_inner;
+    pub use memory_inner;
```

Comment:
This is an auxiliary macro used to implement the memory macro, it shouldn't be made public